pages/static/robots/prod.txt

```diff
@@ -1,4 +1,5 @@
 User-agent: *
 Allow: /
+Disallow: /shared
```

Comment:
We also need to disallow the language specific shared files (see my sample in the issue).

Reply:
They now don't make it to the build anymore with changing the sources for the page transformer: `!${project.paths.GROW_BUILD_DEST}/{*/shared,shared}/*.html`

Sorry, could have mentioned that.

Reply:
clever